sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryManager.scala

```diff
@@ -286,9 +286,7 @@ class StreamingQueryManager private[sql] (sparkSession: SparkSession) extends Lo
 
     (sink, trigger) match {
       case (table: SupportsWrite, trigger: ContinuousTrigger) =>
-        if (operationCheckEnabled) {
-          UnsupportedOperationChecker.checkForContinuous(analyzedPlan, outputMode)
-        }
+        UnsupportedOperationChecker.checkForContinuous(analyzedPlan, outputMode)
```

Comment:
was removing the operationCheckEnabled flag intentional?

Reply:
That's not intentional. Nice finding. Thanks! I'll revert.